dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -749,6 +749,7 @@
         #   index:
         #     prefix: zeebe-record
         #     createTemplate: true
+        #     indexSuffixDatePattern = "yyyy-MM-dd"
```

Comment:
@npepinpe I did here or what do you mean?

Reply:
We could like to https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html ?

Reply:
Since the formatter seems to support several iso formats